.cspell.json

```diff
@@ -16,6 +16,7 @@
   "words": [
     "Abhi",
     "acbs",
+    "acility",
```

Comment:
Should this not be `facility`?

Reply:
Removed as no longer needed (see comment directly below).